cardano-crypto-tests/src/Test/Crypto/Vector/VectorUtil.hs

```diff
@@ -0,0 +1,90 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE DerivingVia #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeFamilies #-}
+
+module Test.Crypto.Vector.VectorUtil
```

Comment:
I'd consider changing the name of this module; it isn't really about vectors at all, it's mostly about hexadecimal representations of bytestrings and keys.

Reply:
Thanks. Yeah. It is more about serialization changed to SerializationUtils.